docs/mallin_lahtotietotiedostot.md

```diff
@@ -0,0 +1,46 @@
+# Mallin lähtötietotiedostot ja niiden muokkaaminen
+
+## HSL:n ylläpitämät lähtötiedot
+
+Yleiskuvaus HSL:n ylläpitämistä aineistoista ja niiden päivitysperiaatteista löytyy [täältä](HSL_lahtotiedot.md). HSL:n lähtötietoaineistot (mm. maankäytöt ja verkkojen tiedot) saat ladattua zip-pakettina, kun olet täyttänyt aineistojen luovutuksen hakemuslomakkeen. Paketti sisältää seuraavat aineistot:
+
+### Ennusteskenaarioiden syöttötiedot
+
+Kansio sisältää kullekin tarkasteluvuodelle/skenaariolle alikansion, jossa on lähtötiedot mm. maankäytön ja kustannukset osalta. **Näihin pitäisi varmaan lisätä readme-selsoteet?**
+
+**Tiedostot**
+
+* **cco** = autoilun kilometrikustannus
+* **edu** = kunkin ennustealueen oppilaspaikkamäärät peruskoulussa, toisella asteella ja korkeakouluissa
+* **ext** = ulkoinen liikenne eli työsäkäyntialueen tienpäät
+* **lnd** = kunkin ennustealueen rakennetun maa-alan osuus sekä erillistalojen osuus rakennuskannasta
+* **pop** = kunkin ennustealueen kokonaisväkiluku sekä eri ikäryhmien osuudet
+* **prk** = kunkin ennustealueen työpaikan ja asiointimatkojen pysäköintikustannukset
+* **tco** = joukkoliikenteen kuukausikustannukset eri vyöhykkeillä
+* **trk** = rekkaliikenteen lähtötiedot: yhdistelmäajoneuvoilta kielletyt ennustealueet ja jätteenkäsittelylaitosten ennustealueet
+* **wrk** = kunkin ennustealueen kokonaistyöpaikkamäärä sekä eri alojen työpaikkojen osuudet (palvelut, kaupat, logistiikka, teollisuus)
+
+### Lähtötiedot
+
+**Täydennettävä selostus**
+
+### Verkot
+
+Kansio sisältää kullekin tarkasteluvuodelle/skenaariolle alikansion, jossa on liikenneverkon lähtötiedot sekä niiden selostus (readme).
+
+**Tiedostot**
+
+* **d211** = liikenneverkon solmut ja linkit sekä niiden attribuutit (HUOM: Helmet 4 käyttää myös solmun label-attribuuttia, jossa lippuvyöhykkeen kirjain)
+* **d221** = joukkoliikennelinjaston kuvaus: mm. linjatunnus, kulkumuoto, ajoneuvotyyppi (vehicle), reitin otsikko, reitin käyttämät solmut (HUOM: pysähtymiset lasketaan solmujen tietojen perusteella verkkojen sisäänajomakrolla (**nimi!!**) ja eri aikajaksojen vuorovälit ajetaan sisään erillisestä tiedostosta)
+* **d231** = kääntymiskiellot eri solmujen välillä
+* **d241_hinta** = linkkikohtainen tienkäyttömaksu eri aikajaksoilla (aht, pt, iht)
+* **d241_pyoratieluokka** = pyörätieluokka eri linkeillä
+* **d241_vuorovalit** = joukkoliikennelinjojen vuorovälit eri aikajaksoilla (aht, pt, iht)
+* **d311_jakoluvut** = *** AIEMPIEN HELMET-VERSIOIDEN LÄHTÖTIEDOSTO (JAKOLUVUT ENN16-->SIJ19, EI TARVITA VERSIOSSA 4
+* **d311_matktermkys** = *** AIEMPIEN HELMET-VERSIOIDEN LÄHTÖTIEDOSTO (LENTOASEMAN JA SATAMIEN MATKAT), EI TARVITA VERSIOSSA 4
+* **d311_tama_ennustevektorit** = *** AIEMPIEN HELMET-VERSIOIDEN LÄHTÖTIEDOSTO (MAANKÄYTTÖVEKTOREITA TAVARALIIKENTEEN MALLIA VAERTEN), EI TARVITA VERSIOSSA 4
+* Muut tiedostot **Tarkistettava näiden yhtenäisyys eri vuosina ja kuvattava mitä sisältyy**
```

Comment:
**Tarkistettava näiden yhtenäisyys eri vuosina ja kuvattava mitä sisältyy**

Reply:
Esiintyykö näitä vielä jakamissamme lähtötiedoissa vai voiko poistaa?

@zptro 
@TimoElolahde